pkgs/development/tools/build-managers/conan_1_xx/default.nix

```diff
@@ -0,0 +1,100 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, git
+, pkg-config
+, python3
+, zlib
+}:
+
+let
+  python = pkgs.python3.override {
+    packageOverrides = self: super: {
+      # conan v1.xx requires old semver instead of node-semver
+      semver = super.semver.overridePythonAttrs (old: rec {
+        version = "0.6.1";
+        src = pkgs.fetchPypi {
+          inherit (old) pname;
+          inherit version;
+          sha256 = "4016f7c1071b0493f18db69ea02d3763e98a633606d7c7beca811e53b5ac66b7";
+        };
+        doCheck = false;
+      });
+    };
+  };
+in
+python.pkgs.buildPythonApplication rec {
+  pname = "conan";
+  version = "1.60.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "conan-io";
+    repo = "conan";
+    rev = "refs/tags/${version}";
+    sha256 = "054l4bv864id1c3pp6plc6af63cbpi9x9xznbqi0zfl8i35ipv3j";
+  };
+
+  nativeBuildInputs = with python.pkgs; [
+    pythonRelaxDepsHook
+  ];
+
+  propagatedBuildInputs = with python.pkgs; [
+    bottle
+    colorama
+    python-dateutil
+    deprecation
+    distro
+    fasteners
+    future
+    jinja2
+    patch-ng
+    pluginbase
+    pygments
+    pyjwt
+    pylint # Not in `requirements.txt` but used in hooks, see https://github.com/conan-io/conan/pull/6152
+    pyyaml
+    semver
+    requests
+    six
+    tqdm
+    urllib3
+  ] ++ lib.optionals stdenv.isDarwin [
+    idna
+    cryptography
+    pyopenssl
+  ];
+
+  pythonRelaxDeps = [
+    # This can be removed once conan is updated to 2.0.7+
+    "PyYAML"
+  ];
+
+  nativeCheckInputs = [
+    git
+    pkg-config
+    zlib
+  ] ++ (with python3.pkgs; [
+    mock
+    parameterized
+    pytest-xdist
+    pytestCheckHook
+    webtest
+  ]);
+
+  __darwinAllowLocalNetworking = true;
+
+  pythonImportsCheck = [
+    "conan"
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Decentralized and portable C/C++ package manager";
+    homepage = "https://conan.io";
+    changelog = "https://github.com/conan-io/conan/releases/tag/${version}";
+    license = licenses.mit;
+    maintainers = with maintainers; [ kuznetsss ];
```

Comment:
You have to add yourself to the maintainer list in a separate commit.

Reply:
Do you mean a separate PR?

Reply:
No, a separate commit is good enough.